### ISSUE TITLE
Actual type not used in generic for constructor parameter

### ISSUE THREAD

**Issue by sethladd** [dart-lang/dartdoc#459]
Consider this code from Sky:

```
...
UINode header;
  UINode content;
  FloatingActionButton fab;
  Drawer drawer;
  List<UINode> overlays;

  Scaffold({
    Object key,
    this.header,
    this.content,
    this.fab,
    this.drawer,
    this.overlays
  }) : super(key: key);
...
```

(https://github.com/domokit/sky_sdk/blob/master/packages/sky/lib/framework/components/scaffold.dart)

Here's the generated docs:

![screen shot 2015-05-07 at 8 51 53 pm](https://cloud.githubusercontent.com/assets/5479/7530320/0e9a8ab4-f4fb-11e4-804b-11f2a48e8f0a.png)

Expected:

`List<UINode> overlays`

Actual:

`List<E> overlays`

**Comment by sethladd** [comment]
Hi @keertip does this look familiar? I seem to remember we had an issue for this already.

**Comment by keertip** [comment]
@sethladd yes, it does look familiar. Thought I fixed an issue similar to this, will take a look at this one.

**Comment by keertip** [comment]
@sethladd  Looks like this has been fixed, don't see this problem when running from head. 

Published new version of package, try dartdoc 0.0.1+10 and reopen if this is still an issue.

**Comment by sethladd** [comment]
Will do! Thanks for triaging.
